codeworld-api/src/CodeWorld/Driver.hs

```diff
@@ -449,14 +456,91 @@ picToObj' pic =
             ptsJS <- pointsToArr pts
             setProps [("points", ptsJS), ("smooth", pToJSVal True)] obj
             retVal obj
-        Path cs pts w closed smooth -> do
-            obj <- init "path"
+        Polygon cs pts -> do
+            obj <- init "polygon"
+            ptsJS <- pointsToArr pts
+            setProps
+                [ ("points", ptsJS)
+                , ("width", pToJSVal True)
```

Comment:
Shouldn't this be 0 instead of True?  Same in 4 other places.

Reply:
I couldn't get it to pass when I was passing 0. That was the w/n Double value I was trying to resolve. It type checked this morning when I passed a bool ie True.

Reply:
Ah, you might need `0 :: Double` because of overloaded number literals and the weird type class.

Reply:
I had tried that and kept getting this :/
![screenshot_2018-05-20_18-42-54](https://user-images.githubusercontent.com/8516378/40285109-174bcf44-5c4d-11e8-8a15-415921fb4c73.png)

Reply:
You have a precedence problem.  The expression `pToJSVal 0 :: Double` is parsed as `(pToJSVal 0) :: Double` (in other words, it asserts that the output of the `pToJSVal` function is a `Double`), when you really meant `pToJSVal (0 :: Double)`.  You need the parentheses to get the latter meaning.

Reply:
Ah. That's really cool! Thanks!